### PR TITLE
French translation improvements from Crowdin made by:

### PR DESCRIPTION
**Pull request by magul**
* Arzugar (https://crowdin.com/profile/arzugar)
* Brice Copy (https://crowdin.com/profile/bcopy)
* Ewen (https://crowdin.com/profile/ewenquim)
* Joachim Jablon (https://crowdin.com/profile/ewjoachim)
* Raphael Das Gupta (https://crowdin.com/profile/das-g)
* Sylvain Chiron (https://crowdin.com/profile/frigory)
* Tom (https://crowdin.com/profile/firgon)

### CODE REVIEW

**Review by ewjoachim — APPROVED** [review]
Whoah that's a good update. I don't agree 100% with all the translations, but it's obviously much better after than before :)

**Comment by magul** [comment]
Do we want second review here @das-g ? Or should we merge it?

File: fr/django_start_project/README.md
Comment (das-g) [inline fr/django_start_project/README.md:116]: If I remember correctly, `{% filename %}Terminal{% endfilename %}` needs to be on a line of its own for the rendering to look right. Let's update the translation now that #1714 has been merged.

(also for the `{% filename %}` occurances below)

**Review by das-g — CHANGES_REQUESTED** [review]

**Comment by das-g** [comment]
> Do we want second review here @das-g ? Or should we merge it?

I think once https://github.com/DjangoGirls/tutorial/pull/1712#discussion_r615257569 is resolved, we can merge. Further improvements can always be made later.

**Comment by das-g** [comment]
I've updated the French translation on Crowdin (using the translations from here, but now hopefully with proper markup).

@magul Please update this PR accordingly. Thanks!

**Comment by magul** [comment]
Done @das-g

**Review by das-g — APPROVED** [review]
Markup looks good now. Regarding language, I'll have to trust the judgement of the native French speakers.

**Pull request merged**
